include/bluetooth/mesh/gen_lvl_srv.h

```diff
@@ -33,6 +33,7 @@ struct bt_mesh_lvl_srv;
 #define BT_MESH_LVL_SRV_INIT(_handlers)                                        \
 	{                                                                      \
 		.handlers = _handlers,                                         \
+		.scene = BT_MESH_SCENE_ENTRY(&_bt_mesh_lvl_srv_scene_type),    \
```

Comment:
What are the arguments for doing it like this instead of making `.scene` a pointer? Seems like we can save 12 bytes of RAM per instance if we do, although the high level models will need 4 extra bytes.
We also wouldn't have to expose the scene type

Reply:
I mean, if it's a pointer, this macro can do `.scene = &((struct bt_mesh_scene_entry){})`, and if higher level models can leave `.scene` alone, and it'll just be a NULL pointer.

Reply:
Ah, I got you. Yeah, sounds good!

Reply:
Then this will still require exposing `scene_type`s. If not do this, compiler won't exclude them and the related code.

Reply:
That's a good point. Sounds like they're worth exposing then.